app/ide-desktop/lib/dashboard/src/components/dashboard/ProjectNameColumn.tsx

```diff
@@ -125,7 +125,7 @@ export default function ProjectNameColumn(props: ProjectNameColumnProps) {
           rowState.setVisibility(Visibility.faded)
           try {
             const createdProject = await backend.createProject({
-              parentDirectoryId: asset.parentId,
+              parentId: asset.parentId,
```

Comment:
Why the rename from `parentDirectoryId` to `parentId`?

Reply:
for consistency... but this is a bug since it's used verbatim in the payload to the Cloud backend. nice catch, thanks!